src/views/ItemAdmin/ItemAdminKPI.vue

```diff
@@ -45,6 +45,30 @@
           </label>
         </validation-provider>
 
+        <hr class="ods-hr" />
+
+        <validation-provider v-slot="{ errors }" rules="required" name="kpiType">
+          <div class="form-group">
+            <span class="form-label">{{ $t('fields.kpitype') }}</span>
+            <span v-if="errors[0]" class="form-field--error">
+              {{ errors[0] }}
+            </span>
+            <div v-for="{ id, label, description } in types" :key="id"
+                 class="ods-form-group descriptive-radio">
+              <input type="radio" :id="id + _uid" class="ods-form-radio"
```

Comment:
Kanskje ikke et stort problem, men bør vurdere bruken av `_uid`? [vue/vuejs](https://github.com/vuejs/vue) issue #5886:
> The vm _uid is reserved for internal use and it's important to keep it private (and not rely on it in user code) so that we keep the flexibility to change its behavior for potential future use cases.